ufo2fontir/src/source.rs

```diff
@@ -50,7 +53,7 @@ impl Cache {
     fn new(
         global_metadata_sources: StateSet,
         glyph_names: HashSet<String>,
-        locations: HashMap<PathBuf, Vec<DesignSpaceLocation>>,
+        locations: HashMap<PathBuf, Vec<UserSpaceLocation>>,
```

Comment:
so yeah that's incorrect, the locations of sources in designspace (in this particular case, `locations` contains the location of each glif file) are in _design_-space coordinates, not user-space

Reply:
My rationale was that for IR generation the userspace location would be what you want?